### DISCUSSION TITLE
Draw UI expansion content over everything else like popup

### DISCUSSION THREAD

**Discussion by dofdary** [zauberzeug/nicegui#2463] [Q&A] [answered]
### Question

Hello,

I have a ui.expansion near the top of my single page application whose content is quite big (a ui.tree that can look like a comb graph sometimes).
When I open it obviously it pushes all the rest of the page content off-screen.
I wonder if there is a way to make it so the content of the expansion is drawn over the widgets below it instead? Like it is the case for ui.select

Thank you in advance if someone has an idea

**Comment by falkoschindler** [comment, marked as answer]
Maybe a [menu](https://nicegui.io/documentation/menu) suits your needs?
The [Quasar documentation](https://quasar.dev/vue-components/menu) can be a good inspiration.

**Reply by dofdary** [reply]
Thanks, I didn't realize menu was so flexible
